app/src/main/java/com/github/se/signify/ui/screens/challenge/ChallengeHistoryScreen.kt

```diff
@@ -22,31 +22,26 @@ fun ChallengeHistoryScreen(
     statsRepository: StatsRepository,
     statsViewModel: StatsViewModel = viewModel(factory = StatsViewModel.factory(statsRepository))
 ) {
-  val friendsChallengesAchieved = statsViewModel.completed.collectAsState()
+  val challengesCompleted = statsViewModel.completed.collectAsState()
   val challengesCreated = statsViewModel.created.collectAsState()
+  val challengesWon = statsViewModel.won.collectAsState()
 
   AnnexScreenScaffold(
       navigationActions = navigationActions,
       testTagColumn = "ChallengeHistoryScreen",
   ) {
-    // Number of friends' challenges achieved
-    StatisticsRow(
-        rowTestTag = "FriendsChallengesRow",
-        lineText = "Number of friends challenges achieved",
-        lineTextTag = "FriendsChallengesText",
-        columnTextList = listOf(listOf("${friendsChallengesAchieved.value}")),
-        columnTextSPList = listOf(listOf(20)),
-        columnTextTagList = listOf("FriendsChallengesCountBox"))
-    Spacer(modifier = Modifier.height(32.dp))
-
-    // Number of challenges created
-    StatisticsRow(
-        rowTestTag = "ChallengesCreatedRow",
-        lineText = "Number of challenges created",
-        lineTextTag = "ChallengesCreatedText",
-        columnTextList = listOf(listOf("${challengesCreated.value}")),
-        columnTextSPList = listOf(listOf(20)),
-        columnTextTagList = listOf("ChallengesCreatedCountBox"))
+    // Challenge's statistics
+    StatisticsColumnRow(
+        columnTestTag = "ChallengesColumn",
+        rowTestTag = "ChallengesRow",
+        lineText = "Number of challenges :",
+        lineTextTag = "ChallengesText",
+        statsTextList = listOf("Completed", "Created", "Won"),
```

Comment:
These hardcoded `String`s would fit well in an enum.

Reply:
It will be put as public static val in stats